components/home/Challenge.tsx

```diff
@@ -1,65 +1,118 @@
-import { PropsWithChildren, useState } from 'react';
-import { StyleSheet, TouchableOpacity, useColorScheme, Dimensions, Image } from 'react-native';
+import { PropsWithChildren, useState } from "react";
+import {
+  StyleSheet,
+  TouchableOpacity,
+  useColorScheme,
+  Dimensions,
+  Image,
+} from "react-native";
 
-import { Colors } from '@/constants/Colors';
-import { ThemedText } from '@/components/theme/ThemedText';
-import { ThemedView } from '@/components/theme/ThemedView';
-import { ThemedIconButton } from '@/components/theme/ThemedIconButton';
+import { Colors } from "@/constants/Colors";
+import { ThemedText } from "@/components/theme/ThemedText";
+import { ThemedView } from "@/components/theme/ThemedView";
+import { ThemedIconButton } from "@/components/theme/ThemedIconButton";
 import { useRouter } from "expo-router";
 
 // Get screen width and height
 const { width, height } = Dimensions.get("window");
 
-export function Challenge({ children, title }: PropsWithChildren & { title: string }) { //image: string
-    const router = useRouter();
+export function Challenge({
+  children,
+  title,
+}: PropsWithChildren & { title: string }) {
+  //image: string
+  const router = useRouter();
 
-    const [isOpen, setIsOpen] = useState(false);
-    const[isLiked, setIsLiked] = useState(false);
-    const theme = useColorScheme() ?? 'light';
+  const [isOpen, setIsOpen] = useState(false);
+  const [isLiked, setIsLiked] = useState(false);
+  const theme = useColorScheme() ?? "light";
 
-    const height = 279; // derived from the height of the image
-    const userName = 'Sandraa'; // derived from the name of the user
-    const userLocation = 'Plage de Vidy'; // derived from the location of the user
-    const userTime = '18:26'; // derived from the time the user posted the challenge
+  const height = 279; // derived from the height of the image
+  const userName = "Sandraa"; // derived from the name of the user
+  const userLocation = "Plage de Vidy"; // derived from the location of the user
+  const userTime = "18:26"; // derived from the time the user posted the challenge
```

Comment:
This is hard-coded for now, you should think of writing this in a more effective manner or keep in mind that it should to stay like this.

Reply:
Yes that's right, these are hard-coded for now, I think we're currently working on implementing the backend of that component with firebase which was not in the scope of my branch.